html/analysis/single_variable/differentiation.html

```diff
@@ -13,7 +13,16 @@
             <div class="definition" id="definition-derivative">
                 <div class="title">derivative</div>
                 <div class="content">
-                    Let `f` be a function and let `a` be a point in its domain, then the derivative of f at a is defined as: `lim_(h->0)(f(a+h)-f(a))/(h)`, when it exists and is denoted by `f^'(a)`
+                    Let `f` be a function and let `a` be a point in its domain, then the derivative of f at a is defined as: `lim_(h->0)(f(a+h)-f(a))/(h)`, when it exists and is denoted by `f^'(a)` or `(d)/(dx)f(a)`
+                </div>
+            </div>
+        </div>
+
+        <div class="thin-wrapper">
```

Comment:
You don't need to make a thin-wrapper for every definition, there is usually only one that wraps the entire page, check out other files to see what I mean